word.js

```diff
@@ -0,0 +1,26 @@
+var Letter = require("./letter.js");
+var fs = require("fs");
+
+var Word = function(){
+	this.size = 0;
+	this.self = function(){
+	var data = fs.readFileSync("secret_words.txt", "utf8");
+	  // Then split it by commas (to make it more readable)
+	  var dataArr = data.replace(/\[|]|"/g, '').split(",");
+
+	  // We will then re-display the content as an array for later use.
+	  var wordList = dataArr;
+	  var letters =  (wordList[(Math.floor(Math.random() * wordList.length))]).split('');
```

Comment:
I wouldn't recommend picking your word this way as it could have the outcome of having to guess the same word again which could lead to a bad user experience.